src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```
@@ -111,7 +112,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="-arch $(CrossCompileArch)" Condition="'$(_IsApplePlatform)' == 'true' and '$(CrossCompileArch)' != ''" />
       <LinkerArg Include="-g" Condition="$(NativeDebugSymbols) == 'true'" />
       <LinkerArg Include="-Wl,--strip-debug" Condition="$(NativeDebugSymbols) != 'true' and '$(_IsApplePlatform)' != 'true'" />
-      <LinkerArg Include="-Wl,-rpath,'$(IlcRPath)'" Condition="'$(StaticExecutable)' != 'true'" />
+      <LinkerArg Include="-Wl,-rpath,$(IlcRPath)" Condition="'$(StaticExecutable)' != 'true'" />
```

```suggestion
      <LinkerArg Include="'-Wl,-rpath,$(IlcRPath)'" Condition="'$(StaticExecutable)' != 'true'" />
```
Would this work better?

---

(and drop the rest of the change)

---

Worth a shot. Thanks!